gm4_note_block_interface/data/nbi/functions/init.mcfunction

```diff
@@ -0,0 +1,22 @@
+#announce module installation
+tellraw @a[gamemode=creative] ["",{"text":"[GM4]: Installing Note Block Interface..."}]
+execute unless entity @p run say GM4: Installing Note Block Interface...
+
+#declare and initialise scoreboards and settings
+scoreboard players set updateHappened gm4_upCheck 1
+scoreboard players set nbi gm4_modules 1
+scoreboard players set nbi gm4_clockTick 0
+
+#announce success
+tellraw @a[gamemode=creative] ["",{"text":"[GM4]: Note Block Interface Installed!"}]
+execute unless entity @p run say GM4: Note Block Interface Installed!
+
+#check other modules to make sure they're up to date.
+execute if score bat_grenades gm4_modules matches ..1 run summon minecraft:area_effect_cloud ~ ~ ~ {CustomName:"\"Bat Grenades\"",Tags:["updateMessage"],Duration:2000}
+execute if score bat_grenades gm4_modules matches ..1 run summon minecraft:area_effect_cloud ~ ~ ~ {CustomName:"\"Another Module\"",Tags:["updateMessage"],Duration:2000}
+
+#scoreboards
```

Comment:
These scoreboards should go up in the scoreboard section of the function to fit the standard.

Reply:
Okay - that is the case with all my modules. I'll sort that later on today 👍